reactive-datatable-template.js

```diff
@@ -23,7 +23,31 @@ Template.ReactiveDatatable.rendered = function() {
         reactiveDataTable.page = info.page;
     });
 
+    // Data difference
+    reactiveDataTable.difference = function(existingData, newData) {
+
+        var targetData = [],
+            existingDataIdArr = _.pluck(existingData, "_id"),
```

Comment:
Also, I thought you removed the reliance on `_id` altogether? I'm not actually sure what the purpose of this function is.